docs/conf.py

```diff
@@ -25,12 +25,17 @@
 # be accessible, and the documentation will not build correctly.
 # See sphinx_astropy.conf for which values are set there.
 
-from datetime import datetime
 import os
 import sys
 
 import configparser
-from pkg_resources import get_distribution
+from datetime import datetime
+from packaging.requirements import Requirement
```

Comment:
Is `packaging` a new dependency too?

Reply:
We are already using it but I see now that it's only for test modules. And as it is a dependency of pytest it's not listed explicitely. It's also a dependency for Sphinx, so it's similar. I'm fine to add it explicitely to `docs` and `tests` if you think that's useful.

```
❯ pip show packaging
...
Required-by: tox, Sphinx, pytest, jupyterlab, jupyterlab-server, jupyterlab-git, jupyter-packaging, build, bokeh, bleach, pytest-doctestplus
```

Reply:
Naw, it's okay. It is a can of worms I don't want to open (listing deps of deps).

Reply:
Well if we use them I think we should list them explicitly. Hypothetically Sphinx or pytest may stop using `packaging` in the future. But that can be for another fight ;)

Reply:
Oh, ops. I merged this PR. If you feel strongly about adding `packaging`, please open a follow up PR. I am neutral on this. Thanks!